lib/src/native/realm_core.dart

```diff
@@ -1,30 +1,49 @@
+////////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2021 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+import 'dart:convert';
 import 'dart:ffi';
-import 'dart:isolate';
-import 'package:ffi/ffi.dart';
-import 'package:ffi/src/utf8.dart';
+import 'dart:typed_data';
+
+// Hide StringUtf8Pointer.toNativeUtf8 and StringUtf16Pointer since these allows to sliently allocating memory. Use toUtf8Ptr instead
+import 'package:ffi/ffi.dart' hide StringUtf8Pointer, StringUtf16Pointer;
 
 import '../configuration.dart';
 import '../realm_class.dart';
+import '../realm_object.dart';
 
 import 'realm_bindings.dart';
 
 late RealmLibrary _realmLib;
 
-
 void setRealmLibrary(DynamicLibrary realmLibrary) {
   _realmLib = RealmLibrary(realmLibrary);
 }
 
 final _RealmCore realmCore = _RealmCore();
 
 class _RealmCore {
-   
   //From realm.h. Currently not exported from the shared library
   // ignore: constant_identifier_names
   static const int RLM_INVALID_CLASS_KEY = 0x7FFFFFFF;
-  // ignore: constant_identifier_names
+  // ignore: constant_identifier_names, unused_field
```

Comment:
consider using ignore_for_file to avoid repetition

Reply:
I intentionally did not do it. since it will ignore everything in the file from now on.